### PR TITLE
Ensure PDF URL is included in DOI registration metadata FOR JOURNALS

### PR DESCRIPTION
**Pull request by myucekul**
The key change is addition of download url for pdf from escholarship. I also noticed that book series template has tabs. Converted tabs to spaces.

### CODE REVIEW

File: logic.py [hunk 1]
```diff
@@ -228,6 +228,10 @@ def get_setting(name, journal):
     return setting_handler.get_setting('plugin:ezid', name, journal).processed_value
 
 def get_journal_metadata(article):
+    # get remote_url from articles
+    itemId = 'qt'+ article.remote_url[-8:]
```
Comment (bibliotechy) [inline logic.py:232]: Should the comment reflect that `remote_url[-8:]` gets the ark, and not the whole remote url?

**Review by bibliotechy — CHANGES_REQUESTED** [review]
Looks good, but I think it makes sense to add a test for this attribute to the test suite.

**Comment by myucekul** [comment]
> Looks good, but I think it makes sense to add a test for this attribute to the test suite.

Good point. I have been using janeway dev for development. Time to build out a dev box and use test suite!

File: logic.py (hunk 1 above)
Comment (myucekul) [reply logic.py:232]: Yes, need updated comment and add a check remote_url before accessing it.

**Review by bibliotechy — APPROVED** [review]

**Pull request merged**